src/test-func/groovy/org/transmartproject/batch/junit/SkipIfJobFailedRule.groovy

```diff
@@ -40,7 +39,9 @@ class SkipIfJobFailedRule implements TestRule {
 
     boolean isJobCompletedSuccessFully() {
         def execution = jobRepositoryProvider().getLastJobExecution(
-                jobName, runJobRule.jobParameters)
+                runJobRule.jobName, runJobRule.jobParameters)
+
+        assert execution != null: 'Could not find execution'
 
         allOf(
```

Comment:
How does it work? Don't you need an actual result to compare with. Something like `assertThat actual, allOf(...`